src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/getfirsttierdocuments/GetFirstTierDocumentsAboutToStartEventHandler.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.getfirsttierdocuments;
+
+import static java.util.Objects.requireNonNull;
+import static org.apache.commons.collections4.CollectionUtils.isEmpty;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class GetFirstTierDocumentsAboutToStartEventHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    @Value("${feature.postHearingsB.enabled}")
+    private final boolean isPostHearingsBEnabled;
```

Comment:
I would imagine that this needs both flags enabled. In practice, B won't be enabled without A also being enabled, so it's probably obsolete to specify both. Maybe good for peace of mind to have both anyway.

Reply:
Thanks, now checking both flags.